Anime-App/Feature/Anime/ViewModel/AnimeViewModel.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  AnimeViewModel.swift
+//  Anime-App
+//
+//  Created by Tobi Adegoroye on 09/08/2023.
+//
+
+import Foundation
+
+class AnimeViewModel {
+    func getAnime(completion: @escaping (Result<[Anime], Error>) -> Void) {
+        // get request
+        guard let url = URL(string: "https://kitsu.io/api/edge/anime") else { return }
+        URLSession.shared.dataTask(with: url) { data, response, error in
```

Comment:
Can you look at creating a new reusable networking manager so you don't have to repeat code for executing a network request 

https://youtu.be/KQlx1rvqV9M